CMakeLists.txt

```diff
@@ -77,29 +77,37 @@ endif()
 #
 option(SAIL_BUILD_APPS "Build applications." ON)
 option(SAIL_BUILD_EXAMPLES "Build examples." ON)
+option(SAIL_BUILD_TESTS "Build tests." ON)
+option(SAIL_BUILD_BINDINGS "Build the C++ and other bindings. Set to OFF if only building C code." ON)
```

Comment:
I like the idea of disabling bindings :+1: I'll take a look.

Reply:
There was no need for creating a new PR. I just moved my changes to a separate branch and reset the usability branch with master. Pushed the binding changes from original PR.

If you have no issue with it, I'd love to go ahead and push the .gitignore addition too since right now at least the tiff binding (and possibly others) creates a test file in the root directory that isn't obvious and could pollute further commits and PRs. The other solution could be going through with a fine tooth cone on the codecs code and fixing it there.

Reply:
Fixed in f7e788a7